draft-ietf-tls-tls13.md

```diff
@@ -1026,14 +1022,12 @@ fragment
 : The AEAD encrypted form of TLSPlaintext.fragment.
 {:br }
 
-Each AEAD cipher suite MUST specify how the nonce supplied to the AEAD
-operation is constructed, and what is the length of the
-TLSCiphertext.nonce_explicit part. In many cases, it is appropriate to use
-the partially implicit nonce technique described in Section 3.2.1 of
-{{RFC5116}}; with record_iv_length being the length of the explicit part. In
-this case, the implicit part SHOULD be derived from key_block as
-client_write_iv and server_write_iv (as described in {{key-calculation}}), and
-the explicit part is included in GenericAEAEDCipher.nonce_explicit.
+The nonce for the AEAD construction is formed by taking the 64-bit
+sequence number (or the 64-bit epoch/sequence number combination
+in the case of DTLS) and padding it on the left with zeroes
+in order to obtain a string of size N_MIN (see {{RFC5116}} Section 4).
+Note: This is a different construction from that in TLS 1.2, which
+specified a partially explicit nonce.
```

Comment:
Note also that an AEAD cipher that has N_MAX less than 64 cannot be used with TLS. ?